### ISSUE TITLE
Cannot load table data outside of MultiDim

### ISSUE THREAD

**Issue by pllim** [ejeschke/ginga#387]
When I attempt to load FITS table data directly either using `File -> Load Image` or `FBrowser` by specifying the desired extension directly in the filename, I get the following traceback:

```
2016-10-17 11:04:53,409 | E | Control.py:532 (load_image) |
    Failed to load file 'j2o1506cj_osc.fits': too many indices for array
Error making callback 'activated': local variable 'image' referenced before assignment
Traceback:
  File ".../ginga/misc/Callback.py", line 98, in make_callback
    res = method(*cb_args, **cb_kwdargs)

  File ".../ginga/gw/GingaGw.py", line 144, in <lambda>
    item.add_callback('activated', lambda *args: self.gui_load_file())

  File ".../ginga/gw/GingaGw.py", line 555, in gui_load_file
    initialdir=initialdir)

  File ".../ginga/qtw/QtHelp.py", line 223, in popup
    self.cb(path)

  File ".../ginga/Control.py", line 653, in load_file
    image.set(loader=image_loader, image_future=future)
```

Perhaps the solution for this will also indirectly address #386?

**Comment by ejeschke** [comment]
See comments in #386.

**Comment by pllim** [comment]
Should be fixed by #397
